jetty-io/src/main/java/org/eclipse/jetty/io/ArrayRetainableByteBufferPool.java

```diff
@@ -390,13 +392,14 @@ private void evict(boolean direct, long excess)
 
                 if (oldestEntry.remove())
                 {
-                    int clearedCapacity = oldestEntry.getPooled().capacity();
+                    RetainableByteBuffer buffer = oldestEntry.getPooled();
+                    int clearedCapacity = buffer.capacity();
```

Comment:
Is it possible for `oldestEntry.getPooled()` to return null?

Reply:
No, `getPooled()` returns `null` only between `reserve()` and `enable()`.
The line before we won the race for `remove()` so we know the entry has been enabled.